index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <!-- <img class="logo" src="images/image_geometry_1.png" alt="logo"> -->
+        <div class="header-img"></div>
```

Comment:
- [ ] [ OPTIONAL ] Good job by implementing the request from the previous review. But, to make your code clean, please kindly remove the commented codes.